#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_pinforc.dpatch by Bas Zoetekouw <bas@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Debian-specific changes to the global pinforc file

@DPATCH@

--- pinfo-0.6.9/src/pinforc.in	2006-03-09 20:44:37.000000000 +0100
+++ pinfo-0.6.9/src/pinforc.in	2009-07-03 11:31:51.815012981 +0200
@@ -82,13 +82,13 @@
 FILTER-0xB7=true
 QUIT-CONFIRMATION=false
 QUIT-CONFIRM-DEFAULT=no
-CLEAR-SCREEN-AT-EXIT=true
+CLEAR-SCREEN-AT-EXIT=false
 CALL-READLINE-HISTORY=true
 STDERR-REDIRECTION="2> /dev/null"
-HTTPVIEWER=lynx
-FTPVIEWER=lynx
-MAILEDITOR=clear; mail
-PRINTUTILITY=lpr
+HTTPVIEWER=sensible-browser
+FTPVIEWER=sensible-browser
+MAILEDITOR=clear; mutt
+PRINTUTILITY=lp
 MANLINKS=1:8:2:3:4:5:6:7:9:n:p:o:3X11:3Xt:3x:3X
 SAFE-USER=nobody
 SAFE-GROUP=nogroup
